README.md

```diff
@@ -53,3 +53,107 @@ python run_bot.py
 - Use [Python v3.6](https://www.python.org/downloads/release/python-360/) as default
 - ssh-keyscan -H <IP address/Hostname> >> ~/.shh/known_hosts
 - sudo -H pip install -U pipenv
+
+## Techstack
+# Programming Language: 
+  -Python 3.6+
+# Libraries:
+  -pipenv: For virtual environment and dependency management
+  -python-telegram-bot: For interacting with Telegram APIs
+  -Reverse Image Search Engines:
+    -Google
+    -Bing
+    -Yandex
+    -TinEye
+    -IQDB
+# Platform:
+  -MS Azure for hosting
+# Databases:
+  -SQLite (for user-specific data like search history, preferences)
+
+### Repository Structure 📂
+
+```plaintext
+ReVot/
+│
+├── .github/                  # GitHub-specific files
+│   └── ISSUE_TEMPLATE/
+│       ├── bug_report.md     # Template for reporting bugs
+│       ├── custom.md         # Custom issue template
+│       └── feature_request.md # Template for requesting features
+│
+├── deploy/                   # Deployment scripts and configurations
+│   ├── after_push            # Post-deployment scripts
+│
+├── reverse_image_search_bot/ # Main bot directory
+│   ├── uploaders/            # Uploader modules
+│   │   ├── __init__.py       # Initialize uploaders package
+│   │   ├── base_uploader.py   # Base class for uploaders
+│   │   ├── file_system.py     # File system operations
+│   │   └── ssh.py            # SSH related functions
+│   │
+│   ├── __init__.py           # Initialize bot package
+│   ├── bot.py                # Main bot logic
+│   ├── commands.py           # Command handling for the bot
+│   ├── image_search.py       # Functions for reverse image search
+│   ├── settings.example.py    # Example settings file for API tokens
+│   ├── settings.example1.py   # Another example settings file
+│   ├── utils.py              # Utility functions and helpers
+│
+├── LICENSE                   # License information
+├── Pipfile                   # Pipenv dependencies
+├── Pipfile.lock              # Locked dependency versions
+├── README.md                 # Main project documentation
+└── run_bot.py                # Script to run the bot
+
+```
+
+# How to Contribute 🌍
```

Comment:
Remove the "How to Contribute" section from `README.md` because it will be covered in `CONTRIBUTING.md`
For now, just refer to this section to `CONTRIBUTING.md`